docs/source/dev_testing.md

```diff
@@ -1,64 +1,45 @@
-# Testing
+# Integration Testing
 
-There are 2 test categories in legion:
-* unit tests, located in `legion/tests/unit`
-* system/integration tests, located in `legion/tests/e2e`
-    * Python tests in `python` sub folder
-    * Robot Framework tests in `robot` sub folder
-    * Stub models in `models` sub folder (are used for Robot Framework tests as payloads)
-
-## Running unit tests
-
-```bash
-make unittests
-```
+Legion uses the robotframework for an integration testing.
+System/integration tests located in the following directories:
+* `legion/robot` - a python package with additional Robot libraries. For example: kubernetes, auth_client, feedback, and so on. 
+* `legion/tests/stuff` - artifacts for integration testing. For example: pre-trained ML artifacts, test toolchain integrations, and so on.
+* `legion/tests/e2e` - directory with the RobotFramework tests.
 
 ## Running system/integration tests
 
-We set up robot tests for `legion-test` cluster in the example below.
-Do not forget change your cluster url and legion version.
-
-* You should meet all requirements for unit tests
-* You should add extra dependencies:
-    1. kops, kubectl binaries
-    2. aws credentials (for calling `kops export`), if you test cluster that is created using kops
-
-* You should export kops context `kops export kubecfg --name legion-test.epm.kharlamov.biz --state s3://legion-cluster` (for kops created cluster) or authorize on cluster using any other mechanism.
-
-* Create the `.secret.yaml` file in the root folder. Fill all value gaps:
-
-```yaml
-dex:
-  enabled: true
-  config:
-    staticPasswords:
-      - email: tests-user@legion.com
-        password: ~
-
-grafana:
-  admin:
-    username: ~
-    password: ~
-```
-
-* Create the `.env` file :
+We set up robot tests for `gke-legion-test` cluster in the example below.
+*Do not forget change your cluster url and legion version.*
+
+Export cluster secrets from legion-profiles directory.
+* Clones [legion-profiles](https://git.epam.com/epmd-legn/legion-profiles) repository. Checkout your or the developer branch.
```

Comment:
I think, there should be no explicit link to internal repo host.

Reply:
Fixed